### PR TITLE
Improved instrumentation to detect long loading times

### PR DESCRIPTION
**Pull request by Larspolo**
image

Might be useful for the timeline bug. Also would also be pretty nice to fix any of these messages

### CODE REVIEW

File: src/vue/src/components/loading/LoadWrapper.vue [hunk 1]
```diff
@@ -41,6 +41,14 @@ export default {
                 }, 500)
             }
         }, 1000)
+        setTimeout(() => {
+            if (this.loading) {
+                this.$store.commit('sentry/CAPTURE_SCOPED_MESSAGE', {
```
Comment (mjvkeulen) [inline src/vue/src/components/loading/LoadWrapper.vue:46]: Do you have an example Sentry event @Larspolo, I would like to check if we need to enrich the context further.

Comment (Larspolo) [reply src/vue/src/components/loading/LoadWrapper.vue:46]: Sentry is down atm, but the request was send

Comment (mjvkeulen) [reply src/vue/src/components/loading/LoadWrapper.vue:46]: Hm weird, is up again

Comment (Larspolo) [reply src/vue/src/components/loading/LoadWrapper.vue:46]: https://sentry.ejournal.app/organizations/ejournal/issues/877/?query=LoadWrapper&statsPeriod=14d

**Review by engelhamer — APPROVED** [review]

File: src/vue/src/components/loading/LoadWrapper.vue (hunk 1 above)
Comment (mjvkeulen) [reply src/vue/src/components/loading/LoadWrapper.vue:46]: We need relevant context, such as the value of `assignment`  and `storeAssignment`, to actually fix the issue?

Comment (mjvkeulen) [reply src/vue/src/components/loading/LoadWrapper.vue:46]: We need a way to split the generated Sentry issues per page (or component)

Comment (Larspolo) [reply src/vue/src/components/loading/LoadWrapper.vue:46]: https://sentry.ejournal.app/organizations/ejournal/issues/983/?project=2&query=&statsPeriod=14d

**Review by mjvkeulen — APPROVED** [review]

**Pull request merged**